pallets/schemas/src/rpc/Cargo.toml

```diff
@@ -10,25 +10,25 @@ repository = "https://github.com/libertyDSNP/frequency/"
 edition = "2021"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.6.1" }
-jsonrpsee = { version = "0.16.2", features = ["client-core", "server", "macros"] }
+codec = { workspace = true }
```

Comment:
```suggestion
parity-scale-codec = { workspace = true }
```

Reply:
This is probably compiled because can actually remove this. Nice catch.